Header header
bool motorsOn
bool calib1
bool calib2

